extra-dev/packages/coq-trakt/coq-trakt.dev/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-trakt"
+maintainer: [ "Enzo Crance <enzo.crance@inria.fr>" "Louise Dubois de Prisque <louise.dubois-de-prisque@inria.fr>" "Chantal Keller <ckeller@lmf.cnrs.fr>" "Tomaz Mascarenhas <tomgm1502@gmail.com>" ]
+authors: [ "Enzo Crance" ]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/ecranceMERCE/trakt"
+bug-reports: "https://github.com/ecranceMERCE/trakt/issues"
+dev-repo: "git+https://github.com/ecranceMERCE/trakt.git"
+doc: "https://ecrancemerce.github.io/trakt/"
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
```

Comment:
I see that the development comes with some Elpi files. Do they need to be installed? Are they?

Reply:
They do not need to be installed. The documentation of Elpi explains that it stores the information in the corresponding .vo file (https://lpcic.github.io/coq-elpi/tutorial_coq_elpi_command.html)